backend/src/seeds/seed.lecture.ts

```diff
@@ -0,0 +1,37 @@
+import { Lecture } from '../lectures/entities/lecture.entity';
+import { Factory } from './seed.utils';
+
+class LectureFactory extends Factory<Lecture> {
+  constructor() {
+    super(Lecture);
+  }
+
+  public generate() {
+    const lecture = new Lecture();
+    lecture.subject = this.faker.lorem.sentence();
+    lecture.body = this.faker.lorem.sentences(10);
+    lecture.index = 0;
+
+    this.entities.push(lecture);
+    return lecture;
+  }
+}
+export const lectureFactory = new LectureFactory();
+
+export const seedLectures = () => [
+  lectureFactory.generateFromData({
+    subject: 'Wykład 1: Podstawy ruchu drogowego',
+    index: 1,
+    body: `<p>Witajcie! Dziś rozpoczynamy kurs na prawo jazdy. W trakcie naszych spotkań nauczymy się podstaw ruchu drogowego oraz przepisów, które należy przestrzegać, aby bezpiecznie poruszać się po drodze.</p><h2>1. Co to jest ruch drogowy?</h2><p>Ruch drogowy to przemieszczanie się pojazdów oraz innych uczestników ruchu po drogach publicznych. W skład uczestników ruchu drogowego wchodzą <a target=""_blank"" rel=""noopener noreferrer nofollow"" href=""http://m.in"">m.in</a>. kierowcy, piesi, rowerzyści, motocykliści oraz pasażerowie.</p><h2>2. Podstawowe zasady ruchu drogowego</h2><p>Podstawową zasadą ruchu drogowego jest bezpieczeństwo. Każdy uczestnik ruchu drogowego powinien zachowywać ostrożność oraz przestrzegać przepisów drogowych. Przepisy te regulują <a target=""_blank"" rel=""noopener noreferrer nofollow"" href=""http://m.in"">m.in</a>. prędkość poruszania się pojazdów, sposób wyprzedzania, zachowanie na skrzyżowaniach oraz obowiązek korzystania z pasów ruchu.</p><h2>3. Znaki drogowe</h2><p>Aby ułatwić poruszanie się po drogach oraz zapewnić bezpieczeństwo, stosuje się znaki drogowe. Znaki te informują o ograniczeniach prędkości, kierunku jazdy, obowiązku zatrzymania się oraz wielu innych kwestiach.</p><h2>4. Prawa i obowiązki kierowcy</h2><p>Kierowcy mają obowiązek przestrzegania przepisów drogowych oraz zachowania ostrożności na drodze. Ponadto, kierowcy muszą posiadać ważne prawo jazdy oraz ubezpieczenie OC. Przepisy drogowe nakładają na kierowców także obowiązek korzystania z pasów ruchu, stosowania sygnałów dźwiękowych oraz zapinania pasów bezpieczeństwa.</p><h2>5. Podsumowanie</h2><p>Dzisiejszy wykład był poświęcony podstawom ruchu drogowego. Nauczyliśmy się, czym jest ruch drogowy oraz jakie są podstawowe zasady, których należy przestrzegać. Omówiliśmy także znaki drogowe oraz prawa i obowiązki kierowcy. W kolejnych wykładach będziemy kontynuować naszą naukę, poznając szczegółowe przepisy drogowe oraz omawiając różne sytuacje, jakie mogą wystąpić na drodze.</p>`,
```

Comment:
ChatGPT for the win!

Reply:
Can't we use an API to generate ALL the course lectures for an entire year? 💯